big_tests/tests/graphql_muc_SUITE.erl

```diff
@@ -295,9 +318,15 @@ end_per_group(Group, _Config) when Group =:= user;
 end_per_group(_Group, _Config) ->
     escalus_fresh:clean().
 
+init_per_testcase(user_shouldnt_store_messages_in_muc_light = TC, Config) ->
+    Config1 = ensure_muc_light_started(Config),
+    escalus:init_per_testcase(TC, Config1);
 init_per_testcase(TC, Config) ->
     escalus:init_per_testcase(TC, Config).
 
+end_per_testcase(user_shouldnt_store_messages_in_muc_light = TC, Config) ->
+    Config1 = ensure_muc_light_stopped(Config),
+    escalus:end_per_testcase(TC, Config1);
```

Comment:
It's better to ensure modules on `init`, and restore on `end` - this way the suite does not change the modules permanently.